src/ls/driver.ts

```diff
@@ -136,7 +138,8 @@ export default class YourDriverClass extends AbstractDriver<DriverLib, DriverOpt
         const queryResultCount = +result.resultSet.numRows
         if (result.resultSet.resultSetHandle !== undefined) {
           const handle = +result.resultSet.resultSetHandle
-          const expectedResults = Math.min(MAX_RESULTS, queryResultCount)
+          const expectedResults = opt['fullResults'] ? queryResultCount : Math.min(MAX_RESULTS, queryResultCount)
```

Comment:
as discussed, this can be accessed with dot notation as well. Not a blocker, though

Reply:
updated